pkgs/development/tools/database/pyrseas/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, python27Packages, fetchFromGitHub }:
```

Comment:
Don't use `python27Packages`. If it works with both 2 and 3, then use `pythonPackages`. If it only works with 2, then `python2Packages`. We don't support `< 2.7` anymore.